src/api/python/cvc5.pxi

```diff
@@ -410,6 +415,7 @@ cdef class DatatypeSelector:
 
 
 cdef class Op:
+    """Wrapper class for :cpp:class:`cvc5::api::Op`."""
```

Comment:
I would prefer to have this more comprehensively documented, so that it is not necessary to pull up the C++ documentation. You can copy over what applies and extend/amend it, but the documentation should be self contained. You can still include the current sentence below the actual documentation, as a reference.

This implies in general, I will not mark all instances.